src/activities/GetCurrentRouteGeometry.ts

```diff
@@ -17,40 +16,50 @@ interface GetCurrentRouteGeometryInputs {
 
     /**
      * Whether to trim the geometry to the current range or return the entire route. Default is false.
-     * 
+     *
      * @displayName Trim To Range
      * @description Whether to trim the geometry to the current range or return the entire route. Default is false.
      */
     trimToRange: boolean;
 }
 
-/** An interface that defines the outputs of the activity. */
 interface GetCurrentRouteGeometryOutputs {
     /**
-     * @description The result of the activity.
+     * The geometry of the current highlighted route.
+     * 
+     * @displayName Geometry
+     * @description The geometry of the current highlighted route.
```

Comment:
Should we also say "or range" on the two lines in this section?

Reply:
Fixed.  Also changed all instances of FeatureSet to Feature Set.  Capitalized to be consistent with descriptions in wf designer.